CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 <a name="unreleased"></a>
 ## [Unreleased]
 
+<a name="v1.1.2"></a>
+## [v1.1.2] - 2022-08-10
+### Bug Fixes
+- **deps:** update rust crate serde_yaml to 0.9.4
+- **deps:** update rust crate pulldown-cmark to 0.9.2
+
+### Features
+- use docker_credential crate instead of DockerConfig
```

Comment:
Maybe add something about building and releasing the Apple Silicon binary?

Reply:
done in https://github.com/kubewarden/kwctl/pull/280/commits/a1b3718aef677acbf98264fd868f59f0f0a82ad5 I wasn't picked up by script probably because the commit was not following the right naming. I manually added it